canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -35,16 +32,26 @@ $toolbar-divider-width: 1px;
 
 	/* Prevent elements from being dragged onto canvas */
 	-webkit-user-drag: none;
+	.toolbar-left-bar
+	.toolbar-item.default.icon_button_1-action button,
```

Comment:
`icon_button_*-action` classnames are specific to the test added in structuretable_paramDef.json for table buttons. These classnames should not be referenced for styling. I also don't see the styles applied in the DOM, are these needed?